manifests/config.pp

```diff
@@ -80,7 +32,7 @@
       group   => 'root',
       mode    => '0644',
       content => template('winbind/oddjobd-mkhomedir.conf.erb'),
-      notify  => Service[['oddjobd', 'winbind',]],
```

Comment:
Need to figure out how to refactor that resource, maybe wrap it in an `if` or something for `$winbind::manage_oddjob_service`, because the init_spec tests under context `with service management disabled` fail since `Service['oddjobd']` is not in the catalog.

See line https://github.com/genebean/genebean-winbind/blob/feature/hiera_5/manifests/service.pp#L14
```puppet
if ($manage_oddjob_service == true) {
  service { 'oddjobd':
    ensure => 'running',
    enable => true,
}
```